src/elements/header/header/readme.md

```diff
@@ -107,6 +107,7 @@ so they were wrapped into a `style` tag and added to the Storybook's configurati
 | `expanded`     | `expanded`       | public  | `boolean`                           | `false` | Whether to allow the header content to stretch to full width. By default, the content has the appropriate page size. |
```

Comment:
I think we need to write more about the logo in size s. Please note that it shouldn't be used on public websites.

Reply:
Logo reverted